src/wp-includes/media.php

```diff
@@ -1843,6 +1843,17 @@ function wp_filter_content_tags( $content, $context = null ) {
 				$filtered_image = wp_img_tag_add_loading_attr( $filtered_image, $context );
 			}
 
+			/**
+			 * Filters an img tag that within the content for a given context.
```

Comment:
```suggestion
			 * Filters an img tag within the content for a given context.
```

I always do this, sorry.